aws/resource_aws_dynamodb_table.go

```diff
@@ -401,7 +402,7 @@ func resourceAwsDynamoDbTableCreate(d *schema.ResourceData, meta interface{}) er
 	}
 
 	if requiresTagging {
-		if err := setTagsDynamoDb(conn, d); err != nil {
+		if err := keyvaluetags.DynamodbUpdateTags(conn, d.Get("arn").(string), []*dynamodb.Tag{}, tags); err != nil {
```

Comment:
Nit: This can use `nil` instead of the custom SDK type 👍 

```suggestion
		if err := keyvaluetags.DynamodbUpdateTags(conn, d.Get("arn").(string), nil, tags); err != nil {
```